modules/find-failing-cis-controls.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-failing-cis-controls_{context}"]
+= Finding failing CIS controls
+
+[role="_abstract"]
+Similar to the *Policy Violations* sunburst chart in the *Configuration Management* view, the CIS controls widget provides information about failing CIS controls.
```

Comment:
Is the "CIS controls" widget a UI object. If so, you could make the name bold. The same applies to other references in the PR, but I won't mention them all in this review.